styles.css

```diff
@@ -2,11 +2,13 @@
 @import url("http://fonts.cdnfonts.com/css/cocogoose");
 
 * {
+  transition: 0.5s;
   font-family: "Lato", sans-serif;
   padding: 0;
   margin: 0;
   box-sizing: border-box;
   color: #272a31;
+  list-style: none;
 }
```

Comment:
Great design @godybuoy 🤝

- [ ] Kindly ensure this section adheres to the design guidelines provided. 
- [ ] You may use the correct font family provided which is `Cocogoose, Lato`.
- [ ] Align the elements on the page proper;y so that they can replicate the design given.
- Your design
![your-main-section](https://user-images.githubusercontent.com/63917725/235324506-63c99b3d-c3ac-487c-af4c-edc4e8a662ce.PNG)
- Given design
![main-section](https://user-images.githubusercontent.com/63917725/235324519-16d2ca76-1ac7-45af-8e27-3f121b011031.PNG)